src/constants.ts

```diff
@@ -43,6 +43,7 @@ export enum APP_PATHS {
   MONITORING = '/monitoring',
   MODULES_FOR_PROJECT = '/modules/project/:projectId',
```

Comment:
Not related to this PR, perhaps we can rehash this a bit?

`/project/{projectId}/modules` for the all modules view
`/project/{projectId}/modules/{modulesId}` for the full modules view

Then, in the contents link, we can use 
`/project/{projectId}/modules/{modulesId}/contents/{contentId}` for any content details or additional resources. 

Perhaps also
`/project/{projectId}/modules/{modulesId}/events/{eventId}` as well for events. If we go this route.

Reply:
If we do stick with `/modules` prefix, (since the rest of our routes do organize like that), another option is to add 

`/modules/project/{projectId}/content/{modulesId}/detail/{contentId}`

to show additional resources or details of module contents.

Reply:
In general, especially for BE API routes, resource based hierarchy does make sense.
For FE routes, my suggestion would be to make them page-view/use-case driven hierarchy. It affects isolation of route-handling code, potential lazy loading, other semantics such as highlighting left nav.
Of course we can make any combination work if there's a strong preference one way or another.